device/dell/x86_64-dellemc_s5232f_c3538-r0/system_health_monitoring_config.json

```diff
@@ -0,0 +1,11 @@
+{
+    "services_to_ignore": [],
+    "devices_to_ignore": ["fan.speed"],
+    "user_defined_checkers": [],
+    "polling_interval": 60,
+    "led_color": {
+        "fault"  : "amber",
+        "normal" : "green",
+        "booting": "blinking_green"
```

Comment:
what does "blinking_amber" indicate?

Reply:
both "amber" and "blinking_amber" represents faulty state.

Reply:
@arunlk-dell does that mean the system_health daemon will set to amber as faulty state even though platform api can set to blinking_amber?

Reply:
yes, system_health daemon checks whether the any of the entity is normal condition or faulty state and the led color is set based on it and for which we have mapped the amber and green color. The blinking_green is for bootup.
But the platform api supports 4 states as the system led is based on the two bit register value.